<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TercihEkrani</class>
 <widget class="QWidget" name="TercihEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tercih Ekranı</string>
  </property>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1730</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>601</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="check_uretici">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ÜRETİCİYİM</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_tuketici">
      <property name="styleSheet">
       <string notr="true">
font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>TÜKETİCİYİM</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
